### PR TITLE
[WIP] Update to KubeVirt v0.12

### PR DESCRIPTION
**Pull request by fabiand**
Signed-off-by: Fabian Deutsch <fabiand@fedoraproject.org>

### CODE REVIEW

**Comment by fabiand** [comment]
This is not yet done, this still needs a traviskube update for ci support.

File: README.md [hunk 1]
```diff
@@ -139,7 +141,7 @@ oc cluster up --skip-registry-check --enable=router,sample-templates
 In addition to the deployment, grant the KubeVirt components some additional roles:
 
 ```bash
-oc adm policy add-scc-to-user privileged -n kube-system -z kubevirt-privileged
-oc adm policy add-scc-to-user privileged -n kube-system -z kubevirt-controller
-oc adm policy add-scc-to-user privileged -n kube-system -z kubevirt-apiserver
+oc adm policy add-scc-to-user privileged -n kubevirt -z kubevirt-privileged
```
Comment (slintes) [inline README.md]: `kubevirt-privileged`, `kubevirt-controller` and  `kubevirt-apiserver` are added to the `privileged` scc by the operator. But we need to do this for `kubevirt-operator` now.

Comment (fabiand) [reply README.md]: +1!

**Comment by fabiand** [comment]
Doh. Obsoleted by #100 and #101

**Pull request closed without merging**